packages/hathor-rpc-handler/src/rpcMethods/sendNanoContractTx.ts

```diff
@@ -20,7 +21,20 @@ import {
   RpcResponse,
   SendNanoContractTxLoadingFinishedTrigger,
 } from '../types';
-import { PromptRejectedError, SendNanoContractTxError } from '../errors';
+import { PromptRejectedError, SendNanoContractTxError, InvalidParamsError } from '../errors';
+import { NanoContractAction } from '@hathor/wallet-lib/lib/nano_contracts/types';
+
+const sendNanoContractSchema = z.object({
+  method: z.string().min(1),
+  blueprint_id: z.string(),
+  nc_id: z.string().nullable(),
+  actions: z.array(z.custom<NanoContractAction>()),
+  args: z.array(z.unknown()).default([]),
+  push_tx: z.boolean().default(true),
+}).refine(
+  (data) => data.blueprint_id || data.nc_id,
+  "Either blueprint_id or nc_id must be provided"
```

Comment:
In the params, `blueprint_id` is not `nullable` nor `optional`, it's just `z.string()`.

Won't this guarantee that it will be provided? Or should we make it optional?

Reply:
We should make it also nullable, but at least one of them should be set (which is what this validation is doing)

Also updated the type for the `sendNanoContractTx`

Thanks